examples/logger.py

```diff
@@ -63,10 +64,12 @@ def __init__(self, parent: Optional[QObject] = None):
         self.logEdit = QTextEdit(self)
         self.logEdit.setReadOnly(True)
         self.clearButton = QPushButton("Clear")
+        self.levelSelector = QComboBox(self)
```

Comment:
Please add this widget into the attributes docstring!

Reply:
Oh, and please rename it to include 'Box`.
For legibilty, we always include the type in each widget's name.